docs/stock/hollow-candlestick-chart.md

```diff
@@ -0,0 +1,16 @@
+Hollow Candlestick chart
+================
+
+A hollow candlestick series is similar to a candlestick series, with a few modifications. The hollow candlestick point is composed of a body and an upper and a lower wick, however, it uses color and fill attributes to show price behavior. 
+
+
+If the current closing price is higher than the opening price (bullish candle) the body is transparent. <br>
+When the current close is lower than the previous close, the line and filling for bearish candles has a reddish color.<br> 
+Else (the close is higher than the previous close), the greenish color is applied for the line and the fill of the bearish candle.
+
+Candles without the fill are called bullish candles and the ones with filling are bearish.
+
+![hollow-candlestick.png](hollow-candlestick.png)
```

Comment:
Navigator gets its' color from the series `color` which is `palette.negativeColor` (red).

Reply:
Hmm.. why `negativeColor` is used, instead of the regular one (picked from `colors` array)?

Reply:
Ah, I see it now (`color: pallete.negativeColor`). It looks really strange to me, however, I'm not sure how we can solve/change it. We have `series.upColor` for bullish candles, so we need to use `series.color` for bearish candles. Maybe simply resign from negative color for bearish candles like we did in a regular candlestick?

Reply:
I guess colours are important in the `hollowcandlestick` and I wouldn't resing from that.
Maybe there should be another API property to describe that colour like `colorBearish`?

Reply:
Right, the best would be to resign from `upColor` and use `bearishColor` (`downColor`?), then the navigator would naturally get what's in `color`. However that will make changing series type harder (between candlestick and hollow candlestick you would also need to manage colors, instead of simple `type` update). 

How about leaving this as it is (navigator color will get red color by default), and adding to the demo `series.navigatorOptions.color = Highcharts.getOptions().colors[0]`? @TorsteinHonsi @karolkolodziej

Reply:
> How about leaving this as it is (navigator color will get red color by default), and adding to the demo series.navigatorOptions.color = Highcharts.getOptions().colors[0]?

This sounds reasonable!